htmengine/htmengine/adapters/datasource/custom/__init__.py

```
@@ -411,7 +411,20 @@ def activateModel(self, metricId):
 
     stats = self._getMetricStatistics(metricId)
 
-    swarmParams = scalar_metric_utils.generateSwarmParams(stats)
+    enableClassifier = False
+    if metricObj.parameters:
+      try:
+        metricParameters = json.loads(metricObj.parameters)
+        if "modelParams" in metricParameters:
+          enableClassifier = metricParameters["modelParams"].get(
+            "enableClassifier", False)
+      except ValueError:
```

@ryanjmccall, so yes, I agree that you need to get rid of this ValueError check, and definitely not suppress errors. :smiley: